### DISCUSSION TITLE
Exo2MediaPlayer error on new SAMSUNG Galaxy Tab A9+ Tablet running Android version 14

### DISCUSSION THREAD

**Discussion by OverDhill** [OpenSageTV/Community#134] [Android client]
Does anyone know how to fix this ExoMediaPlayer error when trying to play back a recorded TV show on the tablet?  I am using Android Miniclient 1.4.4

**Comment by OverDhill** [comment]
No one has any ideas?

**Reply by nyplayer** [reply]
The EXOPLAYER SageTVMiniClient is using is quite old and prone to errors. You might want to remux the recording to try and fix it.

**Comment by LoisGNS** [comment]
I have found that different Android devices work best with different versions of Miniclient. For example, some devices show garbage after the initial start screen on 1.4.4, but if I install an earlier version (I think 1.2.8 tends to work), everything is fine. With other devices, I need to use a different version. So I keep as many versions of the APK on hand as possible until I find the right one for a given device.

**Reply by nyplayer** [reply]
You have to make sure you have the latest SAGETVTRANSCODER plugin installed and the media parser plugin Installed.

**Reply by OverDhill** [reply]
Where do I get the latest SAGETVTRANSCODER plugin (is this for server or  SageTVMiniClient?)?  Also the same question for the  media parser plugin?

**Reply by nyplayer** [reply]
They are server plugins ...

![image](https://github.com/user-attachments/assets/bb0f3159-17cd-4b01-9909-7ab57f2ffe43)

**Comment by gcomrie** [comment]
It looks like there is a slightly newer version of exoplayer available (link to the 2.18.7 amazon port:  https://github.com/amzn/exoplayer-amazon-port). I'm not a developer. Is there anyone who could merge this into the miniclient? I use a few firesticks which is why I pointed to this version.
